src/index.js

```diff
@@ -68,12 +68,11 @@ async function translate(text, options) {
         `${url}?${querystring.stringify(data)}`,
         {
           method: 'POST',
-          body: {
+          body: JSON.stringify({
```

Comment:
Previously, the entire request options was being stringified. But you're only stringifying the body.
Does this work? Have you checked?